addon/ng2/blueprints/component/files/__path__/__name__.ts

```diff
@@ -2,12 +2,10 @@ import {Component} from 'angular2/core';
 
 
 @Component({
+  moduleId: __moduleName,
   selector: '<%= dasherizedModuleName %>',
-  templateUrl: 'app<%= dynamicPath %>/<%= dasherizedModuleName %>/<%= dasherizedModuleName %>.html',
-  styleUrls: ['app<%= dynamicPath %>/<%= dasherizedModuleName %>/<%= dasherizedModuleName %>.css'],
-  providers: [],
-  directives: [],
-  pipes: []
+  templateUrl: '<%= dasherizedModuleName %>.html',
+  styleUrls: ['<%= dasherizedModuleName %>.css']
```

Comment:
Will relative paths work here? How will the web server find the files?
Will they be compiled as part of the component during build?